tasker/job_template/views.py

```diff
@@ -25,16 +20,9 @@ def add_template():
 @bp.route('/template_detail/<id>')
 #@login_required
 def template_detail(id):
-    user = {
-    'username': 'test@testing.com', 'email' : 'test@testing.com', 'timezone' : 'EST', 'view' : 'Month'
-    }
-    templates = [
-    {'id': 0, 'title':'Template 1', 'start_date' : '09.10.2020', 'interval': 3, 'interval_type' : 'Weeks', 'description' : 'Job Template description goes here.'},
-    {'id': 1, 'title': 'Template 2', 'start_date' : '09.15.2020', 'interval': 5, 'interval_type' : 'Days', 'description' : 'Job Template description goes here.'},
-    {'id' : 2, 'title':'Template 3', 'start_date' : '09.20.2020', 'interval' : 1, 'interval_type' : 'Months', 'description' : 'Job Template description goes here.'}
-    ]
-    template = templates[int(id)]
-    return render_template('job-template/template-detail.html', title='Template Details', user=user, template=template)
+    job = JobTemplate.getJobTemplate(id)
```

Comment:
Same comment as above on making sure the job template with the given ID belongs to the user.  This view also needs to `raise NotFound('Job template not found')` if the template with ID does not exist or it exists but belongs to different user.

Reply:
You can check that the job template owner is the current logged in user with current_user (you'll need to test it with a logged in test account):

from flask_login import current_user